src/styles/header.scss

```diff
@@ -0,0 +1,104 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Use Roboto font */
+.body-item {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+/* Header styling */
+.header-item {
+  background-color: #fff;
+  justify-content: space-between;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+/* Logo styling */
+.logo img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+/* Navigation styling */
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  display: flex;
+  justify-content: center; /* Центрування по горизонталі */
+  align-items: center; /* Центрування по вертикалі */
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  display: flex;
+  justify-content: center; /* Центрування по горизонталі */
+  align-items: center; /* Центрування по вертикалі */
+}
+
+.nav__link {
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 60px; /* Додано: Ensuring the link height matches the header height */
+  color: #060b35;
+  text-decoration: none;
+  position: relative;
+  display: inline-block; /* Додано: Ensure the links are treated as blocks */
+  text-align: left;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc; /* Голуба лінія під активним посиланням */
+  position: absolute;
+  left: 0;
+  border-radius: 8px; /* Закруглені краї */
+  margin-bottom: 0;
+  margin-top: -4px;
+}
+
+/* Style for blue text on hover for active link */
+.nav__link.is-active {
+  color: #00acdc; /* Change text color on hover */
+}
```

Comment:
The comment suggests a hover effect, but the selector is for the active state. Ensure that the comment accurately describes the code functionality.